plugins/faustwp/includes/replacement/callbacks.php

```diff
@@ -244,6 +244,24 @@ function enqueue_preview_scripts() {
 	);
 }
 
+add_filter( 'rest_prepare_post', __NAMESPACE__ . '\\preview_link_in_rest_response', 10, 2 );
+add_filter( 'rest_prepare_page', __NAMESPACE__ . '\\preview_link_in_rest_response', 10, 2 );
```

Comment:
Can we add some `has_action` tests for these? They seem like silly tests, but I've had them catch unintentional removals before, so I typically add them.